clusters/ztp-policies/common/group-policies/group-supermicro.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  # yamllint disable-line rule:line-length
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "group-supermicro"
+  namespace: "ztp-group"
+spec:
+  bindingRules:
+    # These policies will correspond to all clusters with this label:
+    group-supermicro: ""
+  mcp: "master"
+  sourceFiles:
+    - fileName: ConsoleOperatorDisable.yaml
+      policyName: "config-policy"
+    - fileName: StorageClass.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "lso-filesystemclass"
+        annotations:
+          storageclass.kubernetes.io/is-default-class: "true"
+    - fileName: DisableSnoNetworkDiag.yaml
+      policyName: "config-policy"
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      spec:
+        additionalKernelArgs:
+          - crashkernel=512M
+          # - cpufreq.default_governor=schedutil 30-Jan testing dcain
+          - rcupdate.rcu_normal_after_boot=0
+        cpu:
+          # These must be tailored for the specific hardware platform
+          isolated: "2-31, 34-63"
```

Comment:
missing reserved

Reply:
Do you aim to have the PerformnaceProfile in the group policies or in site specific?  you should stick to one place.  i would advise you put it in a group specific policy that matches that class of hardware that I ordered (SMC X11) in your case for Goldwing.

Reply:
This is something I wasn't sure about. Should I remove just the `cpu` from the site specific or something more?

Reply:
I think you should remove goldwing-site-specific.yaml entirely, based on looking at group-supermicro.yaml you are duplicating this content.